src/redux/actionTypes/genes.js

```diff
@@ -60,9 +60,25 @@ const MARKER_GENES_LOADED = `${GENES}/markerGenesLoaded`;
  */
 const MARKER_GENES_ERROR = `${GENES}/markerGenesError`;
 
+/**
+ * Downsampled genes expression load task
+ */
+const DOWNSAMPLED_GENES_LOADING = `${GENES}/downsampledGenesLoading`;
+
+/**
+ * Downsampled genes expression loaded
+ */
+const DOWNSAMPLED_GENES_LOADED = `${GENES}/downsampledGenesLoaded`;
```

Comment:
naming is confusing in my opinion: in the platform, when we talk about gene expression we usually have `GENE_EXPRESSION` in the name. Only having `GENES` can confuse that we are talking about the genes from the gene table

Reply:
Good point